client/src/components/App.tsx

```diff
@@ -1,20 +1,51 @@
-import React, { FC } from 'react';
+
+import React, { FC, useState, useEffect } from 'react';
+
+
+
 import Paths from '../Routes';
 import NavigationBar from './NavigationBar';
 import { Routes, Route } from 'react-router-dom';
 import MovieDetail from './MovieDetail';
 
-const App:FC = () => {
+import axios from 'axios';
+import Login from './Login';
+import Profile from './Profile';
+
+
+const App:FC = (props) => {
+  const [currentUser, setCurrentUser] = useState<any>();
+
+  const getLoggedInUser = () => {
+    axios.get('/verify')
+      .then(({data}) => {
+        setCurrentUser(data);
+      })
```

Comment:
need a catch block for error handling

Reply:
Fixed! Currently getting errors in server console on load, as no user set in yet, but it's not crashing the server/page, so you may see what I'm missing.